src/ensdf_processing/HSICC/bldhst.for

```diff
@@ -0,0 +1,131 @@
+C***********************************************************************01 00010
+C*                                                                      01 00020
```

Comment:
@rachelslaybaugh we think we do have permission, correct?
@scopatz common extension?  The extension (.f, .for, etc) varies, I left it in the form it was distributed with.